src/library/tabledelegates/stareditor.cpp

```diff
@@ -83,31 +89,132 @@ void StarEditor::paintEvent(QPaintEvent*) {
         cg = QPalette::Inactive;
     }
 
+    painter.save();
```

Comment:
we have [PainterScope()](https://github.com/mixxxdj/mixxx/blob/d6abcab13affd047ba16640f64e0d0997d1647b2/src/util/painterscope.h#L15) for this

Reply:
Good point, will refactor.